content/posts/newsletter-010/index.md

```diff
@@ -541,6 +541,30 @@ or [join the next meeting][join].
 [abstreet-issues]: https://github.com/dabreegster/abstreet/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
 [mun-issues]: https://github.com/mun-lang/mun/labels/good%20first%20issue
 
+## Jobs
+
+<!-- An optinal section for new jobs related to Rust gamedev -->
+
+- [Elektron][elektron] (synth manufacturer) looking to hire Rust Graphics Engineer
+  (Gotheburg, Sweden):
```

Comment:
Gothenburg is misspelled

Reply:
Copypasted from [TWIR](https://this-week-in-rust.org/blog/2020/05/27/this-week-in-rust-340/). Fixed, thanks!